test/internal/laws/Chain.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var chain = require('./chain_');
+var compose = require('./compose');
+var forall = require('./forall');
+
+
+module.exports = function(equals) {
+  return forall({
+
+    //  m >>= f >>= g = m >>= (f >=> g)
+    associativity: function(m, f, g) {
+      var lhs = chain(g)(chain(f)(m));
+      var rhs = compose(chain(g))(chain(f))(m);
```

Comment:
I've updated `rhs` as you first suggested. I'm reluctant to switch to the three-function version (even though it reads very nicely) as doing so would increase the burden on those using this function to test their data types. Also, the two-function version more closely resembles the expressions in the FL spec.

Do you agree?